contracts/safeguard/guard/GuardedPauser.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity 0.8.17;
+
+import "./Guard.sol";
+
+interface IPauser {
+    function pause() external;
+
+    function unpause() external;
+}
+
+contract GuardedPauser is Guard {
```

Comment:
Maybe explain that we reuse `Guard` as `Pauser`? Upon first glance it's confusing why `GuardedPauser` inherits `Guard` while `GuardedGovernor` doesn't.

Reply:
updated

Reply:
Another option is to merge the governor into this contract as well, so that each chain only needs to have one contract